Assets/Plugin/thirdweb.jslib

```diff
@@ -1,13 +1,17 @@
 var plugin = {
-  ThirdwebInvoke: function (taskId, route, payload, cb) {
+  ThirdwebInvoke: function (taskId, route, payload, callback, cb) {
```

Comment:
i think this should be a separate function, like `ThirdwebAddListener` and `ThirdwebRemoveListener`. It's gonna get messy to mix both one-off invocations vs subscribing to live updates